keyboards/handwired/splittest/bluepill/readme.md

```diff
@@ -0,0 +1,54 @@
+# bluepill bitbang splittest
```

Comment:
I think in general this section could do with a starter paragraph "Make example for this keyboard (after setting up your build environment):"

Reply:
Does the updated documentation explains the parts you were missing ?